pages/SetPasswordPage.js

```diff
@@ -0,0 +1,161 @@
+import React, { useState } from 'react';
+import { StyleSheet, View, Linking, Alert } from 'react-native';
+import {Button, Text } from 'galio-framework';
+import { CustomInputPassword } from '../components/CustomInput';
+import { validateSetPassword, showAlertForInvalidInput } from '../components/Validation';
+import { setToken, getToken } from '../components/TokenManager';
+import qs from 'qs';
+
+const SetPasswordPage = (props) => {
+
+  const { navigation } = props;
+  const [ user, setUser] = useState({});
+  const [ errors, setErrors] = useState({});
+
+  const handleInput = (value, name) => {
+      setUser({ ...user, [name]: value})
+      console.log(user)
+  }
+
+  const extractTokenFromURL = () => {
+    console.log("In extract token")
+    Linking.getInitialURL().then((url) => {
+      if (url) {
+        console.log('Initial url is: ' + url); 
+
+        //url1 = "app://boxapp/setpassword?confirm=NPKZ9qxgWypV7Qmei1eP"
+        let urlArray=url.split("?")
+        let queryParams = urlArray[1]
+        let setPasswordToken = qs.parse(queryParams).confirm
+        console.log(setPasswordToken)
```

Comment:
remove console.log.

Add comment what are these set of lines doing with some example.